src/utils.js

```diff
@@ -146,16 +147,46 @@ export function getSelectionAffectedTableCells( selection ) {
  *
  *		console.log( `Selected rows ${ first } to ${ last }` );
  *
- * @package {Array.<module:engine/model/element~Element>}
+ * @param {Array.<module:engine/model/element~Element>}
  * @returns {Object} Returns an object with `first` and `last` table row indexes.
  */
 export function getRowIndexes( tableCells ) {
-	const allIndexesSorted = tableCells.map( cell => cell.parent.index ).sort();
+	const indexes = tableCells.map( cell => cell.parent.index );
 
-	return {
-		first: allIndexesSorted[ 0 ],
-		last: allIndexesSorted[ allIndexesSorted.length - 1 ]
-	};
+	return getFirstLastIndexesObject( indexes );
+}
+
+/**
```

Comment:
A couple of documentation issues here. Wording, wrong param name, lack of param name in the `@param` line, etc. I'm going to fix that on master.

Reply:
Thanks!